geocity/settings.py

```diff
@@ -598,3 +615,13 @@
         ],
     },
 }
+
+
+def show_toolbar(request):
+    """Shows the debug toolbar when `?DEBUG=true` is your URL and DEBUG is enabled."""
+    return DEBUG and strtobool(request.GET.get("DEBUG", "false"))
```

Comment:
Use a condition instead of strtobool to keep the same structure as other settings.
`request.GET.get("DEBUG", "false").lower() == "true"`

Or should we adapt everything to use strtobool, to keep the same logic everywhere

Reply:
Ok